manager/integration/tests/test_basic.py

```diff
@@ -210,8 +236,13 @@ def test_snapshot(clients, volume_name):  # NOQA
 def snapshot_test(client, volname):
     volume = client.by_id_volume(volname)
 
+    start_pos = VOLUME_RWTEST_START_POSITION
     snap1 = volume.snapshotCreate()
+    snap2_wdata = generate_random_data(VOLUME_RWTEST_COUNT)
+    volume_write(volume["endpoint"], start_pos, snap2_wdata)
     snap2 = volume.snapshotCreate()
+    snap3_wdata = generate_random_data(VOLUME_RWTEST_COUNT)
+    volume_write(volume["endpoint"], start_pos, snap3_wdata)
```

Comment:
After snap3 deleted, snap3_wdata should still exist. Deletion of a snapshot shouldn't affect the data of volume.

The data written for snap1/snap2/snap3 need to be in the different positions to verify that all the previous data still intact, even after purge, if no revert happened.